src/Nethermind/Nethermind.JsonRpc/Modules/Eth/GasPriceConfig.cs

```diff
@@ -0,0 +1,16 @@
+using Nethermind.Int256;
+
+namespace Nethermind.JsonRpc.Modules.Eth
+{
+    public static class GasPriceConfig
+    {
+        public const int NoHeadBlockChangeErrorCode = 7;
```

Comment:
Add comments explaining what these settings mean For example -> Percentile can be percentile of anything